src/Pages/PackageSelector/Components/Banner.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+const Banner = ({ children, bgcolor, color, radius, weight }) => {
```

Comment:
```suggestion
import styled from "styled-components";

const Banner = ({ children, bgColor, color, radius, weight }) => {
```
- import와 컴포넌트 선언 사이 한줄 띄워주세요
- `bgColor` 카멜 케이스 적용해주세요